### PR TITLE
Bug 1791194: Removing KubeVersionMismatch alert

### PR DESCRIPTION
**Pull request by paulfantom**
Backporting #611 as per comments in https://bugzilla.redhat.com/show_bug.cgi?id=1751706

/cc @LiliC @s-urbaniak

### CODE REVIEW

File: assets/grafana/dashboard-definitions.yaml [hunk 1]
```diff
@@ -64,7 +64,6 @@ items:
                       "threshold": 0
                   },
                   "datasource": "prometheus",
-                  "description": "",
```
Comment (LiliC) [inline assets/grafana/dashboard-definitions.yaml]: Do we think these things would cause any problems?

Comment (paulfantom) [reply assets/grafana/dashboard-definitions.yaml]: No, those shouldn't case issues.

**Comment by paulfantom** [comment]
rebased

**Comment by paulfantom** [comment]
rebased
/bugzilla refresh

**Comment by sdodson** [comment]
@paulfantom Needs a rebase.

**Comment by sdodson** [comment]
/lgtm cancel
Since rebase will invalidate lgtm anyway

**Comment by lilic** [comment]
@paulfantom needs rebase and magical regenerate. :)

**Pull request merged**